📖clusterctl based quick-start (#2590)
kubernetes-sigs/cluster-api: docs/book/src/reference/glossary.md

# T
---

> ### Target management cluster

Reviewer: I've seen people use the word "target cluster" to talk about the workload cluster, a.k.a. not the management cluster. What does "target management cluster" mean? The definition below is a little unclear to me, it seems like it can be both the management cluster and the workload cluster depending on the situation...

Reviewer 2: Originally "Target" was introduced to differentiate the pivoting use case specifically, rather than trying to use the existing management cluster/workload cluster type terminology which doesn't really hold up for pivoting use cases (where one could pivot to or from a workload cluster, or in some use cases to/from might not even be a workload cluster).

The idea was to use Source/Target to talk about the pivot workflow instead. This clarification was my attempt to try and disambiguate the usage a bit. Since target only applies to pivoting, then it is only applicable to management clusters and not any other type of cluster we discuss.

I'm open to any other way we can disambiguate these concepts a bit more though, because we haven't been overly successful to date :joy:

Reviewer: Could we just remove the terminology "target" from all docs then? And instead just refer to "management" and "workload" clusters everywhere?

Reviewer 2: sgtm

Author: IMO we can remove this definition from the glossary, but I defer this to @detiber

Author: done!

The existing Kubernetes cluster we intend to use as a Cluster API management cluster by installing Cluster API

> and it's related components by running `clusterctl init`.

Reviewer: ```suggestion
and its related components by running `clusterctl init`.
```

> ### Target cluster

Reviewer 2: ```suggestion
### Target management cluster
```

> The declared cluster we intend to create and manage using Cluster API when running `clusterctl init`.

Reviewer 2: ```suggestion
The existing Kubernetes cluster we intend to use as a Cluster API management cluster by installing Cluster API and it's related components by running `clusterctl init`.
```

Reviewer 3: *its 😄

> When using a [Bootstrap Cluster](#bootstrap-cluster) this refers to the workload-cluster that will be the new management cluster.

Reviewer 2: ```suggestion
When using a [Bootstrap Cluster](#bootstrap-cluster) this refers to the [Workload Cluster](#workload-cluster) that will be the new management cluster.
```
